api/v1alpha1/ratelimit_types.go

```diff
@@ -91,6 +91,37 @@ type RateLimitRule struct {
 	// 429 HTTP status code is sent back to the client when
 	// the selected requests have reached the limit.
 	Limit RateLimitValue `json:"limit"`
+	// UsageSpecifier specifies the "usage" of each request that matches the rule.
+	// The usage is used to reduce the rate limit counters for the matching requests.
+	// The reduction happens after the request stream is complete, so the rate limit
+	// won't be enforced for the current request, but for the subsequent requests.
+	//
+	// This is optional and if not specified, the rate limit counters are not reduced.
```

Comment:
Not sure if I understand this correctly, but the default behaivor of "usage of each request" is one? We may need to rephrase this comment to make it more understandable.